artmaker/src/main/java/io/artmaker/ArtMakerViewModel.kt

```diff
@@ -174,64 +141,46 @@ internal class ArtMakerViewModel(
     }
 
     private fun selectStrokeWidth(strokeWidth: Int) {
-        preferences.set(
-            key = PREF_SELECTED_STROKE_WIDTH,
-            value = strokeWidth,
-        )
+        preferencesManager.updateStrokeWidth(strokeWidth = strokeWidth)
         _uiState.update {
             it.copy(
-                strokeWidth = preferences.get(
-                    PREF_SELECTED_STROKE_WIDTH,
-                    defaultValue = 5,
-                ),
+                strokeWidth = preferencesManager.getStrokeWidth(strokeWidth = strokeWidth),
             )
         }
     }
 
     private fun updateStylusSetting(useStylusOnly: Boolean) {
-        preferences.set(PreferenceKeys.PREF_USE_STYLUS_ONLY, useStylusOnly)
+        preferencesManager.updateStylusOnlySetting(useStylusOnly = useStylusOnly)
         _uiState.update {
             it.copy(
-                shouldUseStylusOnly = preferences.get(
-                    key = PreferenceKeys.PREF_USE_STYLUS_ONLY,
-                    false,
-                ),
+                shouldUseStylusOnly = preferencesManager.getStylusOnlySetting(useStylusOnly = useStylusOnly),
```

Comment:
The default values of the preferences should not come from the value being used to set the preferences. Use the default values that were with those preferences before. In the case of stylus setting, eg, the default value was false. the getters from the pref manager do not need to have any paraemeters